src/sparseml/evaluation/integrations_config.yaml

```diff
@@ -14,3 +14,4 @@
 # my_eval_integration: integrations/my_eval_integration.py
 # my_eval_integration2: integrations/my_eval_integration2.py
 
+perplexity: integrations/perplexity.py
```

Comment:
would be good to have any sort of coverage for this to protect against files being moved since we're referencing relative path

Reply:
Added as a part of #2043